utils/rpms/daos.spec

```diff
@@ -0,0 +1,227 @@
+# Needed because of the GO binaries
+%undefine _missing_build_ids_terminate_build
+
+%define daoshome %{_exec_prefix}/lib/%{name}
+
+Name:          daos
+Version:       0.4.0
+Release:       2%{?dist}
+Summary:       DAOS Storage Engine
+
+License:       Apache
+URL:           https//github.com/daos-stack/daos
+Source0:       %{name}-%{version}.tar.gz
+Source1:       scons_local-%{version}.tar.gz
+
+BuildRequires: scons
+BuildRequires: cart-devel
+BuildRequires: argobots-devel >= 1.0rc1
+BuildRequires: libpmem-devel, libpmemobj-devel
+BuildRequires: fuse-devel >= 3.4.2
+BuildRequires: protobuf-c-devel
+BuildRequires: spdk-devel, spdk-tools
+BuildRequires: fio < 3.4
+BuildRequires: libisa-l-devel
+BuildRequires: raft-devel
+BuildRequires: mercury-devel
+BuildRequires: openpa-devel
+BuildRequires: libfabric-devel
+BuildRequires: openssl-devel
+BuildRequires: ompi-devel
+BuildRequires: pmix-devel
+BuildRequires: hwloc-devel
+BuildRequires: libevent-devel
+BuildRequires: libyaml-devel
+BuildRequires: libcmocka-devel
+BuildRequires: readline-devel
+%if (0%{?rhel} >= 7)
+BuildRequires:  numactl-devel
+BuildRequires: CUnit-devel
+BuildRequires: golang-bin
+BuildRequires: libipmctl-devel
+%else
+%if (0%{?suse_version} >= 1315)
+BuildRequires:  libnuma-devel
+BuildRequires: cunit-devel
+BuildRequires: go1.10
+BuildRequires: ipmctl-devel
+%endif
+%endif
+Requires: cart
+Requires: argobots >= 1.0rc1
+Requires: libpmem, libpmemobj
+Requires: fuse >= 3.4.2
+Requires: protobuf-c
+Requires: spdk
```

Comment:
won't this pull in the most recent version? I think we are tied to a specific version

Reply:
It could, potentially, if any distro we are currently packaging for were shipping spdk packages.  But since they are not (TTBOMK) we just end up getting the one we package.

That said, we are quite a ways behind with spdk.  Probably nearly a year now.  It seems it's probably way past time to get up to a more recent release.